transport/websocket/websocket.go

```
@@ -87,7 +85,6 @@ func (s *Socket) writerRoutine() {
 		}
 
 		if err := s.socket.WriteJSON(msg); err != nil {
-			log.Println("error writing into websocket", err)
 		}
```

Simply ignore the error